### PR TITLE
fix(wing): move wing to new repo

### PR DESCRIPTION
**Pull request by MarkMcCulloh**
Thanks for dealing with the annoying constant updates from https://github.com/winglang/wing 🙏 

https://github.com/winglang/tree-sitter-wing should now only be updated when there is a change to the grammar (still sourced from the wing monorepo), and all the files are already committed so there should be no need to `generate`.

Also updated the queries/tests to match the current state of the repo

### CODE REVIEW

**Review by clason — APPROVED** [review]

**Comment by clason** [comment]
You could also run `scripts/update-readme.lua`.

**Comment by lucario387** [comment]
Missing a lockfile bump it seems

**Comment by clason** [comment]
Yeah; our compilation CI is ... not great.

File: README.md [hunk 1]
```diff
@@ -437,7 +437,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [vue](https://github.com/tree-sitter-grammars/tree-sitter-vue) (maintained by @WhyNotHugo, @lucario387)
 - [x] [wgsl](https://github.com/szebniok/tree-sitter-wgsl) (maintained by @szebniok)
 - [x] [wgsl_bevy](https://github.com/theHamsta/tree-sitter-wgsl-bevy) (maintained by @theHamsta)
-- [x] [wing](https://github.com/winglang/wing) (experimental, maintained by @gshpychka, @MarkMcCulloh)
+- [x] [wing](https://github.com/winglang/tree-sitter-wing) (experimental, maintained by @gshpychka, @MarkMcCulloh)
```
Comment (clason) [inline README.md:440]:
```suggestion
- [x] [wing](https://github.com/winglang/tree-sitter-wing) (maintained by @gshpychka, @MarkMcCulloh)
```

Comment (MarkMcCulloh) [reply README.md:440]: It's currently flagged as experimental in parsers.lua, should this still be removed?

Comment (clason) [reply README.md:440]: Oh, sorry, I _meant_ in parsers.lua.

**Comment by clason** [comment]
thanks!

**Comment by lucario387** [comment]
If you don't accept wing when it's update spamming you don't deserve wing when it's no longer monorepo ❤️

**Pull request merged**